src/hooks/useContract.ts

```diff
@@ -335,14 +335,14 @@ export function useLPTokenContract(
 }
 
 export function useGaugeControllerContract(): GaugeController | null {
-  const { chainId } = useActiveWeb3React()
+  const { account, chainId } = useActiveWeb3React()
   const contractAddress = chainId
     ? GAUGE_CONTROLLER_ADDRESSES[chainId]
     : undefined
   return useContract(
     contractAddress,
     GAUGE_CONTROLLER_ABI,
-    false,
+    account ? true : false,
```

Comment:
Is account necessary here?

Reply:
Good question. I think we can give `undefined` for this parameter.
But I am not exactly sure why we used `false` instead of `undefined`

Reply:
Does anything change if an account is not included in this?

Reply:
Now, we are having `withSignerIfPossible` to give `account` to `getContract`.
We can get account from `useActiveWeb3React`, it means we don't need ` withSignerIfPossible` parameter on `useContract` to check.
On the contract returning from useContract, we can check whether given contract is containing `signer` or not.

Also, useContract is using on  to create specific contract, so removing this parameter will make more clean.